src/components/FollowersFollowing.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { Button } from "antd";
+import FollowersFollowingUsers from "./FollowersFollowingUsers";
+import { preFabProfiles } from "../test/testProfiles";
+
+const FollowersFollowing = (): JSX.Element => {
+  enum ListStatus {
+    CLOSED = "",
+    FOLLOWERS = "showFollowers",
+    FOLLOWING = "showFollowing",
+  }
+
+  const [selectedListTitle, setSelectedListTitle] = useState(ListStatus.CLOSED);
+
+  const tempUserList = [
+    {
+      profile: preFabProfiles[0],
+      icon: preFabProfiles[0].icon,
+      name: preFabProfiles[0].name,
+      following: true,
+      followsMe: true,
+    },
+    {
+      profile: preFabProfiles[1],
+      icon: preFabProfiles[1].icon,
+      name: preFabProfiles[1].name,
+      following: true,
+      followsMe: false,
+    },
+    {
+      profile: preFabProfiles[2],
+      icon: preFabProfiles[2].icon,
+      name: preFabProfiles[2].name,
+      following: false,
+      followsMe: true,
+    },
+  ];
```

Comment:
I would use the stored graph data in Redux instead of creating it here. Then, when we switch to the SDK the datatypes will match up and it will still work as opposed to requiring changes to the Profile to hook it up.|

There are a few changes that will be needed for the redux store to make sure we populate correctly with mock data though. Like right now we load in a list of people you follow and are following but we aren't grabbing their profiles. It'd make sense to load up the profile of everyone in your graph in the background. 

Let me know if you want to pair working on that.